bin/ksql-run-class

```diff
@@ -16,7 +16,7 @@ fi
 # create logs directory
 if [[ -n $LOG_DIR ]]; then
     [[ -d $LOG_DIR ]] || mkdir -p "$LOG_DIR"
-    KSQL_LOG4J_OPTS="-Dksql.log.dir=$LOG_DIR ${KSQL_LOG4J_OPTS}"
+#    KSQL_LOG4J_OPTS="-Dksql.log.dir=$LOG_DIR ${KSQL_LOG4J_OPTS}"
```

Comment:
Is this intentional?

Reply:
Yes. Had to comment it to get the build going.

Reply:
But that doesn't seem like the right fix. The `log4j-rolling.properties` file used by the ksql server refers to this variable: https://github.com/confluentinc/ksql/blob/master/config/log4j-rolling.properties#L8

Where would the logs go if this line is commented out? Why does it need to be commented out?

Reply:
The logs go to the path that is set by CP. You will be able to see the logs the same way you see the cp logs:
`confluent log ksql`